storm-core/test/clj/org/apache/storm/metrics_test.clj

```diff
@@ -376,26 +376,5 @@
       (assert-metric-running-sum! "mybolt" "__ack-count/myspout:default" 2 4 cluster)
       (assert-metric-running-sum! "mybolt" "__execute-count/myspout:default" 3 4 cluster))))
 
-(deftest test-system-bolt
```

Comment:
Do we want to add some unit tests in java code?

Reply:
I don't think it's worth the effort for getting this in.  If you feel replacing this test is important enough, I would suggest opening a new JIRA.